pkg/cli/flags.go

```diff
@@ -539,7 +539,9 @@ func init() {
 	clientCmds = append(clientCmds, nodeCmds...)
 	clientCmds = append(clientCmds, systemBenchCmds...)
 	clientCmds = append(clientCmds, nodeLocalCmds...)
+	clientCmds = append(clientCmds, userFileCmds...)
 	clientCmds = append(clientCmds, stmtDiagCmds...)
+	clientCmds = append(clientCmds, userFileCmds...)
```

Comment:
i think you somehow doubled this line in a rebase since you also have an extra commit